src/MoBi.Presentation/Repositories/MenuBarItemRepository.cs

```diff
@@ -234,7 +234,7 @@ protected override IEnumerable<IMenuBarItem> AllMenuBarItems()
             .WithDescription(ToolTips.ModelingRibbon.CreateModule)
             .WithIcon(ApplicationIcons.Module);
 
-         yield return CreateMenuButton.WithCaption(AppConstants.Captions.ExpressionProfile)
+         yield return CreateSubMenu.WithCaption(AppConstants.Captions.ExpressionProfile)
```

Comment:
I don't quite understand how this is done here. 
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Presentation/Repositories/MenuBarItemRepository.cs#L178

this is the original code in PKSim. Why is is done different in MoBI, I do not know. But the  "AddMetaEnzyme" etc sub entries needs to be added to the main menu. Which I don't see here

Reply:
Why is that sub-menu from PK-Sim built that way, compared to:

https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/develop/src/PKSim.Presentation/Repositories/MenuBarItemRepository.cs#L383

Where it's created in ButtonGroupRepository? What's the difference between them?